.eslintrc.json

```diff
@@ -16,15 +16,19 @@
                 "sourceTag": "*",
                 "onlyDependOnLibsWithTags": ["*"]
               }
-            ]
+            ],
+            "allowCircularSelfDependency": true
```

Comment:
Just curious, why did we add the `allowCircularSelfDependency`, have our code happen the circular self-dependency, or just to add for preventing it happened in the future, or other reason ?

Reply:
NX linter blocks the relative imports from self if this value = false, e.g. "@vulcan-sql/core/model" from core package.
https://github.com/nrwl/nx/blob/master/packages/workspace/src/tslint/nxEnforceModuleBoundariesRule.ts#L184

Reply:
Oh, I see, thanks for sharing the source code 🤣  and replying my question 👍